perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1870,6 +1874,32 @@ sub ban_user : Private {
     return 1;
 }
 
+sub unban_user : Private {
+    my ( $self, $c, $user ) = @_;
+
+    if ($user->email_verified && $user->email) {
+        my $abuse = $c->model('DB::Abuse')->find({ email => $user->email });
+        if ( $abuse ) {
+            $abuse->delete;
+            $c->stash->{status_message} = _('user removed from abuse list');
+        } else {
+            $c->stash->{status_message} = _('user not in abuse list');
+        }
+        $c->stash->{username_in_abuse} = 0;
+    }
+    if ($user->phone_verified && $user->phone) {
+        my $abuse = $c->model('DB::Abuse')->find({ email => $user->phone });
+        if ( $abuse ) {
+            $abuse->delete;
+            $c->stash->{status_message} = _('user removed from abuse list');
+        } else {
+            $c->stash->{status_message} = _('user not in abuse list');
+        }
+        $c->stash->{username_in_abuse} = 0;
+    }
+    return 1;
```

Comment:
Given you removed an unused return 1 further down, don't think this one is used.